pkg/numaresourcesscheduler/manifests/yaml/configmap.nrt.yaml

```diff
@@ -13,8 +13,25 @@ data:
       - schedulerName: topo-aware-scheduler
         plugins:
           filter:
+            disabled:
+              - name: '*'
             enabled:
+              - name: NodeUnschedulable
+              - name: NodeName
+              - name: TaintToleration
+              - name: NodeAffinity
+              - name: NodePorts
+              - name: NodeResourcesFit
               - name: NodeResourceTopologyMatch
```

Comment:
Please keep me honest here but the original proposal (i.e. the sorted one)  was to have ```NodeResourceTopologyMatch``` prior to ```NodeResourcesFit```  is it?

https://github.com/openshift-kni/numaresources-operator/pull/286/commits/c03f5cb2105a0196b364d715169d3616628023e3#diff-18edee9604a942c68e91625bd378355cdaf900fb7f3bb0c9363f01d0981e5b8fR38

Reply:
correct. I'm changing it intentionally, I'll update the commit message to provide the rationale.